cmd/fluxctl/install_cmd.go

```diff
@@ -28,74 +22,32 @@ func newInstall() *installOpts {
 func (opts *installOpts) Command() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "install",
-		Short: "Bootstrap Flux, installing it in the cluster and initializing its manifests in the Git repository",
-		RunE:  opts.RunE,
+		Short: "Print and tweak Kubernetes manifests needed to install Flux in a Cluster",
+		Example: `# Install Flux and make it use Git repository git@github.com:<your username>/flux-get-started
+fluxctl install --git-url 'git@github.com:<your username>/flux-get-started' | kubectl -f -`,
+		RunE: opts.RunE,
 	}
-	cmd.Flags().StringVarP(&opts.gitURL, "git-url", "r", "",
+	cmd.Flags().StringVarP(&opts.gitURL, "git-url", "", "",
 		"URL of the Git repository to be used by Flux, e.g. git@github.com:<your username>/flux-get-started")
-	cmd.Flags().StringVarP(&opts.gitBranch, "git-branch", "b", "master",
+	cmd.Flags().StringVarP(&opts.gitBranch, "git-branch", "", "master",
 		"Git branch to be used by Flux")
-	cmd.Flags().StringVarP(&opts.gitLabel, "git-label", "l", "flux",
-		"Git label to keep track of FLux's sync progress; overrides both --git-sync-tag and --git-notes-ref")
-	cmd.Flags().StringVarP(&opts.gitFluxPath, "git-flux-subdir", "p", "flux/",
-		"Directory within the Git repository where to commit the Flux manifests")
-	cmd.Flags().DurationVarP(&opts.timeout, "timeout", "t", 10*time.Second,
-		"Timeout duration for I/O operations")
-	cmd.Flags().StringVarP(&opts.namespace, "namespace", "n", "flux",
+	cmd.Flags().StringVarP(&opts.gitLabel, "git-label", "", "flux",
+		"Git label to keep track of Flux's sync progress; overrides both --git-sync-tag and --git-notes-ref")
+	cmd.Flags().StringVarP(&opts.gitUser, "git-user", "", "Flux",
+		"Username to use as git committer")
+	cmd.Flags().StringVarP(&opts.gitEmail, "git-email", "", "flux-dev@googlegroups.com",
```

Comment:
This should be mandatory, rather than defaulting.

Reply:
Would it make sense to default this to what's in the user'd the git configuration (if found)?

Reply:
Don't think it would, most of the time this is set to the email address / account of some GitHub bot.